vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -66,8 +66,11 @@
         AVAILABLE("available"),
         @SerializedName("uploading")
         UPLOADING("uploading"),
+        @SerializedName("transcode_starting")
+        TRANSCODE_STARTING("transcode_starting"),
         @SerializedName("transcoding")
```

Comment:
This is a little nitpicky, but since we are in this space, I suggest using constants in the SerializedName and constructor params, and for all Status items.